### PR TITLE
Slow down creation of additional pods in density.go e2e

### PR DESCRIPTION
**Pull request by piosz**

### CODE REVIEW

**Comment by piosz** [comment]
cc @davidopp @fgrzadkowski @gmarek

File: test/e2e/density.go [hunk 1]
```diff
@@ -286,14 +284,18 @@ var _ = Describe("Density", func() {
...
-					Name:         additionalRCName,
-					Namespace:    ns,
-					PollInterval: itArg.interval,
-					Replicas:     minionCount,
+				// Create some additional pods with throughput ~5 pods/sec.
+				var wg sync.WaitGroup
+				wg.Add(minionCount)
+				for i := 1; i <= minionCount; i++ {
+					name := additionalPodsPrefix + "-" + strconv.Itoa(i)
+					labels := map[string]string{
```
Comment (gmarek) [inline test/e2e/density.go]: This can probably be moved  outside of the loop.

**Comment by gmarek** [comment]
LGTM, short of small nit.

File: test/e2e/density.go (hunk 1 above)
Comment (piosz) [inline test/e2e/density.go]: Done

**Comment by satnam6502** [comment]
@k8s-bot retest this please

**Comment by satnam6502** [comment]
Issue? Risk Level? Milestone? Thanks.

**Comment by satnam6502** [comment]
Since this is a test I shall merger it without an associated milestone.

**Comment by piosz** [comment]
Thanks @satnam6502. 

Issue is measuring performance in general. Risk level is low since it's only changing the test and actually the logic being changed is disabled by default as for now.

File: test/e2e/density.go [hunk 2]
```diff
@@ -286,14 +284,18 @@ var _ = Describe("Density", func() {
...
+				// Create some additional pods with throughput ~5 pods/sec.
+				var wg sync.WaitGroup
+				wg.Add(minionCount)
+				podLabels := map[string]string{
+					"name": additionalPodsPrefix,
+				}
+				for i := 1; i <= minionCount; i++ {
+					name := additionalPodsPrefix + "-" + strconv.Itoa(i)
+					go createRunningPod(&wg, c, name, ns, "gcr.io/google_containers/pause:go", podLabels)
+					time.Sleep(200 * time.Millisecond)
```
Comment (bprashanth) [inline test/e2e/density.go]: Nit: if you're trying to rate limit to 5qps you can use util.throttle

**Pull request merged**